blocks/carousel/carousel.css

```diff
@@ -115,4 +115,43 @@ main .carousel-dot-button {
 main .carousel-dot-button.selected {
     background-color: #cacaca;
     border: 0;
-}
\ No newline at end of file
+}
+
+/* hero slider stying */
+main .hero-slider .carousel-item {
+    position: relative;
+    aspect-ratio: 16/9; /* for hero slider */
+}
+
+main .hero-slider .carousel-item-image {
+    position: absolute;
+}
+
+/* hero slider text */
+main .hero-slider .carousel-item-text {
+    position: absolute;
+    z-index: 1;
+    background-color: rgb(0 0 0 / 50%);
+    min-height: 188px;
+    padding: 0 28px;
+    bottom: 0;
+    left: 0;
+}
+
+/* hero slider text */
+main .carousel-item-text p {
+    margin: 10px 0 16px;
+    line-height: 1.2;
+    font-weight: 400;
+    color: white;
+}
+
+/* hero slider text */
+main .carousel-item-text h2 {
```

Comment:
would it make sense to prefix these as well with `.hero-slider`? otherwise they will apply to all carousels you have as well